test/Numerics/ODESolvers/ode_tests_basic.jl

```diff
@@ -118,7 +118,7 @@ Qexact = exactsolution(finaltime, q0, t0)
         errors = similar(dts)
         for (method, order) in imex_methods
             for split_explicit_implicit in (false, true)
-                for variant in (LowStorageVariant(), NaiveVariant())
+                for variant in (NaiveVariant(),)
```

Comment:
Oops, we commented out `LowStorageVarianet()` to speed up our local debugging. We need to restore this and add check somewhere to skip the case when `LowStorageVariant()` and Trap2 appear together here.

Reply:
Okay, thanks @thomasgibson . I will add this back. Where should we check for this case and Trap2 to appear together?

Reply:
Don't worry, I will push a fix to this --- I'm just testing a few things locally.

Reply:
Thanks!